src/pages/index.js

```diff
@@ -0,0 +1,68 @@
+import React from "react"
+import { FormattedMessage, useIntl } from "gatsby-plugin-intl"
+// import { Link } from "gatsby"
```

Comment:
🤮

Reply:
Aw crap, yeah ... missed that. I can fix that.